app/src/main/java/com/codesoom/assignment/TaskHandler.java

```diff
@@ -0,0 +1,95 @@
+package com.codesoom.assignment;
+
+import com.codesoom.exception.TaskNotFoundException;
+import com.codesoom.http.HttpMethod;
+import com.codesoom.http.HttpRequest;
+import com.codesoom.http.HttpResponse;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+
+import static com.codesoom.assignment.HttpStatus.BAD_REQUEST;
+import static com.codesoom.assignment.HttpStatus.CREATED;
+import static com.codesoom.assignment.HttpStatus.NOT_FOUND;
+import static com.codesoom.assignment.HttpStatus.NO_CONTENT;
+import static com.codesoom.assignment.HttpStatus.OK;
+
+public class TaskHandler implements HttpHandler {
+    private static final int PLACE_OF_TASK_ID_FROM_PATH = 2;
+
+    private final TaskService taskService = new TaskService();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        try {
+            handleRequest(exchange);
+        } catch (Throwable throwable) {
+            throwable.printStackTrace();
+            throw throwable;
+        }
+    }
+
+    private void handleRequest(HttpExchange exchange) throws IOException {
+        HttpResponse httpResponse = new HttpResponse(exchange);
+
+        if (!HttpRequest.isValidRequest(exchange)) {
+            httpResponse.response(BAD_REQUEST, "유효하지 않은 Http 요청입니다.");
+            return;
+        }
+        HttpRequest httpRequest = new HttpRequest(exchange);
```

Comment:
이런 건 final을 붙여주면 좋겠죠.
```suggestion
        final HttpRequest httpRequest = new HttpRequest(exchange);
```

Reply:
안녕하세요 종립님

final 을 여기에 붙이는 이유가 무엇일까 궁금합니다.
로컬 변수에 final 을 붙이면 재할당을 막을 수 있는데, 제가 봤을 때는 메서드의 body 가 짧아서 재할당될 여지가 거의 없어 보입니다.
만약 붙인다면 body가 긴 serviceTask() 메서드의 path, method 로컬변수에 final을 붙이면 적합하지 않을까 싶습니다.

혹시 추후에 바디가 길어질 것을 대비한 것일까요?